pulsar/src/main/java/com/flipkart/varadhi/pulsar/services/PulsarTopicService.java

```diff
@@ -35,4 +43,31 @@ public PulsarStorageTopic get(String topicName) {
         throw new NotImplementedException();
     }
 
+    @Override
+    public void delete(PulsarStorageTopic topic) {
+        try {
+            clientProvider.getAdminClient().topics().deletePartitionedTopic(topic.getName(), true, true);
```

Comment:
let both of them be false for now.
No schema currently.
let delete fail if there is active producer or subscription connected to it.